### PR TITLE
Re arrange docs for Ops

### PR DESCRIPTION
**Pull request by oke-aditya**
Well let's categorize the information. It will make it easier to understand.
Open to ideas :smiley: 

cc @NicolasHug

### CODE REVIEW

File: docs/source/ops.rst [hunk 1]
```diff
@@ -10,49 +10,98 @@ Operators
...
+    :template: class.rst
+
+    FeaturePyramidNetwork
+    MultiScaleRoIAlign
+    RoIAlign
+    RoIPool
+
+
+Box Operators
+~~~~~~~~~~~~~
```
Comment (oke-aditya) [inline docs/source/ops.rst:37]: Maybe add one line description for each?

File: docs/source/ops.rst [hunk 2]
```diff
@@ -10,49 +10,98 @@ Operators
 .. note::
   All operators have native support for TorchScript.
```
Comment (oke-aditya) [inline docs/source/ops.rst:12]: On line 8. Should we add more comprehensive summary?

**Review by oke-aditya — COMMENTED** [review]
Thoughts?

File: docs/source/ops.rst (hunk 2 above)
Comment (datumbox) [reply docs/source/ops.rst:12]: Yes that's worth it. Perhaps mention that this included layers, losses etc.

File: docs/source/ops.rst (hunk 1 above)
Comment (datumbox) [reply docs/source/ops.rst:37]: Sure, as long as we have something meaningful and helpful to say that doesn't repeat what we say on the summary above.

File: docs/source/ops.rst [hunk 3]
```diff
@@ -10,49 +10,98 @@ Operators
...
 
+
+Detection Operators
```
[the remark this answers is not in the document]
Comment (oke-aditya) [reply docs/source/ops.rst]: Yep merged them :smiley:

**Comment by oke-aditya** [comment]
I have tried adding one line descriptions. Please have a look :)

**Comment by oke-aditya** [comment]
Rendered docs are here 
https://output.circle-artifacts.com/output/job/6db8e8ad-f465-4de3-827d-b2dbc1e38563/artifacts/0/docs/ops.html

File: docs/source/ops.rst [hunk 4]
```diff
@@ -5,54 +5,97 @@ Operators
...
+    distance_box_iou_loss
+    generalized_box_iou_loss
     sigmoid_focal_loss
-    stochastic_depth
+
+
+Layers
+~~~~~~
+
+Torchvision provides commonly used building blocks as layers which can be composed together to create models.
```
Comment (datumbox) [inline docs/source/ops.rst:79]:
```suggestion
TorchVision provides commonly used building blocks as layers:
```

**Pull request merged**